src/components/Home/MetricsCards.tsx

```diff
@@ -76,6 +75,7 @@ const MetricsCards = () => {
       description={t('page.home.metrics.ending_soon', { value: endingSoonProposals?.total })}
     />,
     <MetricsCard
+      href="/#engagement"
```

Comment:
Just thinking out loud: would it make sense to make this id a shared constant?

Reply:
Not sure. Maybe if we start doing this repeatedly we could have an `AnchorHashes` object.
```
enum AnchorHash = {
  Engagement = "engagement"
}
```
And do something like...
```href={`/#${AnchorHash.Engagement`}``` and `id={AnchorHash.Engagement}`